backend-transit/src/main/java/eu/socialedge/hermes/backend/transit/domain/Route.java

```diff
@@ -64,7 +64,7 @@ public Route(String code, VehicleType vehicleType, List<Station> stations, Shape
         this.vehicleType = notNull(vehicleType);
         this.stations = new ArrayList<>(notEmpty(stations));
 
-        if (!containsAllStations(shape))
+        if (!containsAllStations(notNull(shape)))
```

Comment:
Shape is optional in GTFS too btw
https://developers.google.com/transit/gtfs/reference/trips-file